### ISSUE TITLE
QuickUnlock and root entry location

### ISSUE THREAD

**Issue by rangerxusa** [KN4CK3R/KeePassQuickUnlock#9]
Hi,
You have created a great and useful plugin for KeePass. 

I was wondering if it was possible to support allowing the QuickUnlock entry to be in other folder locations than just the root of the KeePass database. I like to keep the root empty so I can click it and clear the entry list from prying eyes. It would be nice to be able to store the QuickUnlock entry anywhere.

I can make cosmetic changes to plugin code, but your code is beyond my capabilities.

Thanks for considering this request! I hope you keep this plugin updated to work with future versions of KeePass.

**Comment by KN4CK3R** [comment]
How do you image the alternative? I have chosen the root folder because it is empty in my database too and so I don't see the entry all the time. Should the user tell the plugin which is the QuickUnlock entry (I don't like that) or should the plugin scan the whole database for the QuickUnlock entry?

**Comment by rangerxusa** [comment]
Thanks for the quick reply. All ways have pros and cons of course.

For the most flexibility, I think search all groups starting with the root and looking for the first entry that has an exact title of QuickUnlock is what I'm thinking. That way the entry can be hidden (or at least not so obviously exposed as it is in the root. Considering how important that entry is (it can unlock the database) in my opinion it is best not to be so conspicuous.
I think KeePass is a great tool, but I don't like that when you open the database it goes to the last edited entry and displays that group's list. If someone is looking over your shoulder that is way to much information exposed. I got around that somewhat by setting a trigger on open database (and when I click on a customized button) to list entries with tag clear and I set up one dummy entry with a tag of clear. At least now on open only that one entry is displayed. I wish KeePass had an option on open to always go to the Root.

But I also like to click on the empty Root Group when working in KeePass to hide all entries... at least until now with it showing the QuickUnlock entry. And as I mentioned earlier, that is a very critical entry.

Thanks for considering this request.
